APM > .NET Tracer > .NET Core Setup (#11387)
DataDog/documentation: content/en/tracing/setup_overview/setup/dotnet-core.md

## Installation and getting started

### Configure the Datadog Agent for APM

> Install and configure the Datadog Agent to receive traces from your now instrumented application. By default the Datadog Agent is enabled in your `datadog.yaml` file under `apm_config` with `enabled: true` and listens for trace traffic at `localhost:8126`. For containerized environments, follow the links below to enable trace collection within the Datadog Agent.

Reviewer: "to receive traces from your now instrumented application" sounds very forced grammatically, and doesn't make sense anymore if we configure the Agent before the Tracer.

Reviewer: "in" vs "within"

```suggestion
Install and configure the Datadog Agent to receive traces from your now instrumented application. By default the Datadog Agent is enabled in your `datadog.yaml` file under `apm_config` with `enabled: true` and listens for trace traffic at `localhost:8126`. For containerized environments, follow the links below to enable trace collection in the Datadog Agent.
```

{{< tabs >}}

{{% tab "Containers" %}}

1. Set `apm_non_local_traffic: true` in the `apm_config` section of your main [`datadog.yaml` configuration file][1].

2. See the specific setup instructions to ensure that the Agent is configured to receive traces in a containerized environment:

{{< partial name="apm/apm-containers.html" >}}
</br>

> 3. After having instrumented your application, the tracing client sends traces to `localhost:8126` by default.  If this is not the correct host and port change it by setting the below env variables:

Reviewer: ```suggestion
3. After instrumenting your application, the tracing client sends traces to `localhost:8126` by default.  If this is not the correct host and port change it by setting the below env variables:
```

>     `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT`.

Reviewer: Should we use `DD_TRACE_AGENT_URL` here? The plan some time ago was to move towards the single `DD_TRACE_AGENT_URL` instead of these two, so `DD_TRACE_AGENT_URL` overrides them if they are present. This can be confusing for users if they follow this doc.

Author: I'll double check, but yeah if this is the direction we're going, I can post a separate PR to do the cross-language change.

    See [configure the tracer](#configure-the-tracer) for more information on how to set these variables.
{{< site-region region="us3,eu,gov" >}} 

4. Set `DD_SITE` in the Datadog Agent to {{< region-param key="dd_site" code="true" >}} to ensure the Agent sends data to the right Datadog location.

{{< /site-region >}}

[1]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
{{% /tab %}}

{{% tab "AWS Lambda" %}}

To set up Datadog APM in AWS Lambda, see the [Tracing Serverless Functions][1] documentation.

[1]: /tracing/serverless_functions/
{{% /tab %}}

> {{% tab "Azure App Services Extension" %}}

Reviewer: ```suggestion
{{% tab "Azure App Service" %}}
```

{{< /tabs >}}

> ### Configure the Datadog Agent for APM

Author: The diff looks confusing but I moved the "Configure the Datadog Agent for APM" section so it is now the last subsection of "Installation and getting started" and I also updated the text to match the other languages
